mac/Focalboard/AutoSaveWindowController.swift

```diff
@@ -6,6 +6,7 @@ import Cocoa
 class AutoSaveWindowController: NSWindowController {
 
 	override func windowDidLoad() {
+        self.window?.tabbingMode = .disallowed
```

Comment:
You can actually set this in the storyboard (see the drop-down in the lower right corner of the screenshot below). Setting it in code should have the same effect but since there are already lots of other settings applied in the storyboard, keeping it there could simplify future maintenance.

![Screenshot 2021-05-02 at 20 57 38](https://user-images.githubusercontent.com/1137962/116824178-28e68f80-ab89-11eb-9062-a31f384901df.png)

Reply:
@Johennes, that's convenient, thanks!